README.md

```diff
@@ -78,7 +78,7 @@ Options:
 -i, --indexPrefix string   Index prefix to search (default "ind")
 -r, --bearerToken string   Bearer token of your org to ingest (default "")
 -n, --numIterations int    Number of iterations to send query suite (default 10)
--f, --filePath string      path to csv file containing query suite to send to server. Expects CSV of with [search text, startTime, endTime, indexName, relation, count, queryLanguage] in each row
+-f, --filePath string      path to csv file containing query suite to send to server. Expects CSV of with [search text, startTime, endTime, indexName, evaluation type, relation, count, queryLanguage] in each row
```

Comment:
now that the evaluation type is more complex, maybe we can add more information in the read me as to the structure.

We can also add the examples on the pull request readme to this readme